app/src/main/java/org/eyeseetea/malariacare/SplashScreenActivity.java

```diff
@@ -67,6 +68,9 @@ private void init() {
             splashActivityStrategy.initPullFilters(pullFilters);
             splashActivityStrategy.executePull(pullUseCase, pullFilters);
         }
+
+        SurveyDB.deleteOlderSentSurveys();
```

Comment:
As explained before this is in the main, so I think we would need to either change the gradle var to the convenient variant, or create a strategy to create a performMaintenanceTasks method that could be altered depending on the variant.